simulation_ws/src/cloudwatch_simulation/launch/small_house_turtlebot_navigation.launch

```diff
@@ -14,31 +14,57 @@
   <!-- If true and "follow_route" is false, will follow a dynamic route forever -->
   <arg name="dynamic_route" default="false"/>
 
-  <!-- 
+  <!--
        Always set GUI to false for AWS RoboMaker Simulation
        Use gui:=true on roslaunch command-line to run with a gui.
   -->
   <arg name="gui" default="false"/>
 
   <!-- World and Robot -->
-  <include file="$(find aws_robomaker_small_house_world)/launch/small_house_turtlebot_navigation.launch">
+  <!-- Turtlebot robot model -->
+  <arg name="model" default="$(optenv TURTLEBOT3_MODEL waffle_pi)" doc="model type [burger, waffle, waffle_pi]"/>
+
+  <!--
+      Only allow users to set (x,y,yaw), otherwise initial
+      2D pose estimates will be wrong
+  -->
+  <arg name="x_pos" default="3.5"/>
+  <arg name="y_pos" default="1.0"/>
+  <arg name="yaw" default="0.0"/>
+
+  <!-- Load the small house world into Gazebo -->
+  <include file="$(find aws_robomaker_small_house_world)/launch/small_house.launch">
     <arg name="gui" value="$(arg gui)"/>
-    <arg name="x_pos" value="3.5"/>
-    <arg name="y_pos" value="1.0"/>
-    <arg name="yaw"   value="0.0"/>
+  </include>
+
+  <!-- Spawn the robot into Gazebo with the turtlebot description -->
+  <include file="$(find turtlebot3_description_reduced_mesh)/launch/spawn_turtlebot.launch">
+    <arg name="x_pos" default="$(arg x_pos)"/>
+    <arg name="y_pos" default="$(arg y_pos)"/>
+    <arg name="yaw" default="$(arg yaw)"/>
+  </include>
+
+  <!-- Enable navigation system -->
+  <include file="$(find cloudwatch_simulation)/launch/turtlebot3_navigation.launch">
```

Comment:
this can directly reference to [`(find turtlebot3_navigation)/launch/turtlebot3_navigation.launch`](https://github.com/ROBOTIS-GIT/turtlebot3/blob/master/turtlebot3_navigation/launch/turtlebot3_navigation.launch)

Reply:
As mentioned in another comment, I think we need the modified version of `turtlebot3_navigation.launch` because of the configurable initial pose, which is not present in the turtlebot3_navigation package.